### DISCUSSION TITLE
v3.15.6

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1894] [Announcements]
### Bug Fixes

* **autocmds:** only get buffer infromation on BufEnter event ([b17ca3c](https://github.com/AstroNvim/AstroNvim/commit/b17ca3c324aec72ccb9053f2de32e472276d9ce5))

### Reverts

* remove telescope workaround ([76429f5](https://github.com/AstroNvim/AstroNvim/commit/76429f5c1d0c69e476245f9caa2ffd8bb2ae2b73))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.15.5...v3.15.6

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.15.6'>v3.15.6</a>.</em>

**Comment by patrickphat** [comment]
How to move between tabs?
